jenkins/CBDeps.cmake

```diff
@@ -0,0 +1,78 @@
+IF (NOT CBDeps_INCLUDED)
```

Comment:
Most of this file is copied from a couple different places in tlm, with some minor tweaks.